extensions/wikia/WikiaHubsServices/modules/WikiaHubsModuleService.class.php

```diff
@@ -90,8 +91,8 @@ protected function getView($viewName, $data) {
 		return $this->app->getView(get_class($this), $viewName, $data);
 	}
 
-	protected function getImageInfo($fileName, $destSize = 0) {
-		return ImagesService::getLocalFileThumbUrlAndSizes($fileName, $destSize, ImagesService::EXT_JPG);
+	protected function getImageInfo($fileName) {
+		return GlobalFile::newFromText($fileName, $this->cityId);
```

Comment:
Nice. I didn't know `GlobalFile`. It probably works better than `ImagesService` which worked some time ago :smile_cat: 
And it seems it existed back when we worked on hubs services... what a pity we didn't know it.